Source/ChocolateyGui.Common.Windows/Services/ChocolateyService.cs

```diff
@@ -497,8 +495,8 @@ public async Task<ChocolateySource[]> GetSources()
             // we need to read all information from the config file, i.e. the username and password
             var config = await GetConfigFile();
             var allSources = config.Sources.Select(_mapper.Map<ChocolateySource>).ToArray();
-
-            var filteredSourceIds = _configSettingsService.source_list(_choco.GetConfiguration()).Select(s => s.Id).ToArray();
+            var configSettingsService = new ChocolateyConfigSettingsService(_xmlService);

```

Comment:
What was the reason that this is being done, rather than relying on the injected version of the service?